### PR TITLE
unicorn(-emu): 1.0.2-rc4 -> 1.0.2

### PR DESCRIPTION
**Pull request by luc65r**
###### Motivation for this change
Updated unicorn and simplified the file.
Since the `unicorn` attribute name is free for some time now, I used it

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by luc65r** [comment]
@thoughtpolice could you take a look?

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -1649,7 +1649,9 @@ in
   candle = libsForQt5.callPackage ../applications/misc/candle { };
 
   capstone = callPackage ../development/libraries/capstone { };
-  unicorn-emu = callPackage ../development/libraries/unicorn-emu { };
+
+  unicorn = callPackage ../development/libraries/unicorn { };
+  unicorn-emu = unicorn;
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:1654]: Please move that to ``pkgs/top-level/aliases.nix``.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102085` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>7 packages failed to build:</summary>
  <ul>
    <li>pwndbg</li>
    <li>pwntools (python38Packages.pwntools)</li>
    <li>python27Packages.pwntools</li>
    <li>python27Packages.unicorn</li>
    <li>python37Packages.pwntools</li>
    <li>python37Packages.unicorn</li>
    <li>python38Packages.unicorn</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>unicorn (unicorn-emu)</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102085` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>7 packages failed to build:</summary>
  <ul>
    <li>pwndbg</li>
    <li>pwntools (python38Packages.pwntools)</li>
    <li>python27Packages.pwntools</li>
    <li>python27Packages.unicorn</li>
    <li>python37Packages.pwntools</li>
    <li>python37Packages.unicorn</li>
    <li>python38Packages.unicorn</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>unicorn (unicorn-emu)</li>
  </ul>
</details>

**Comment by luc65r** [comment]
IDK why license was bsd3, it is clearly stated on the website that it is licensed under gpl2

**Comment by luc65r** [comment]
Now everything should work @SuperSandro2000

**Comment by luc65r** [comment]
now ofborg-eval-package-list-no-aliases should be happy

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102085` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages failed to build:</summary>
  <ul>
    <li>pwntools (python38Packages.pwntools)</li>
    <li>python27Packages.pwntools</li>
    <li>python37Packages.pwntools</li>
  </ul>
</details>
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python27Packages.unicorn</li>
    <li>python37Packages.unicorn</li>
    <li>python38Packages.unicorn</li>
    <li>unicorn</li>
  </ul>
</details>

**Comment by risicle** [comment]
`pwntools` appears already broken on darwin in master. Same result for me on macos 10.14.

On linux x86_64, all rebuilds are happy.

**Comment by luc65r** [comment]
I guess it can be merged in this state since it doesn't break what wasn't broken before.
Someone who can test on darwin could try to fix `pwntools` in another PR.

**Review by mkg20001 — APPROVED** [review]
LGTM

**Pull request merged**